docs/content/quickstart/quick-start-docker.md

```diff
@@ -1,9 +1,19 @@
-<!-- vale off -->
-# Quickstart w. Docker
 
```

Comment:
Remove this empty line

Reply:
@sabakhilji there's an empty line at the very beginning of the documentation please remove it